src/api/authentication.js

```diff
@@ -217,5 +225,42 @@ export async function authenticate (ctx, next) {
   return next()
 }
 
+export async function getEnabledAuthenticationTypes (ctx, next) {
+  // Log level for messages logged
+  let logLevel = 'error'
+
+  try {
+    if (!authorisation.inGroup('admin', ctx.authenticated)) {
+      ctx.statusCode = 403
+      const error = `User ${ctx.authenticated.email} is not an admin, API access to get enabled authentication types denied.`
```

Comment:
@bradsawadye This const here is unnecessary as it's only used once.

Reply:
addressed in https://github.com/jembi/openhim-core-js/pull/1082/commits/562dc5c4c469b99a04afcc0d09a9d002436aac15